app/grandchallenge/algorithms/templates/algorithms/job_detail.html

```diff
@@ -172,25 +172,22 @@ <h2>Algorithm Result</h2>
                     <dd class="col-sm-9">
                         <ul class="list-unstyled mb-0">
                             {% for output in object.outputs.all %}
-                                {% for file in output.image.files.all %}
-                                    <li>
-                                        <a class="badge badge-primary"
-                                           href="{{ file.file.url }}">
-                                            <i class="fa fa-download"></i>
-                                            Output {{ forloop.parentloop.counter }} {{ file.image.name|stem }}
-                                            ({{ file.file|suffix }})
-                                        </a>
-                                    </li>
-                                {% endfor %}
                                 {% if output.file %}
-                                    <li>
-                                        <a class="badge badge-primary"
-                                           href="{{ output.file.url }}" download="{{ output.file.name|name }}">
-                                            <i class="fa fa-download"></i>
-                                            Output {{ forloop.parentloop.counter }} {{ output.file.name|stem }}
-                                            ({{ output.file|suffix }})
-                                        </a>
-                                    </li>
+                                  <li>
+                                    <a class="badge badge-primary"
+                                       href="{{ output.file.url }}">
+                                        <i class="fa fa-download"></i> {{ output.interface.title }}
+                                    </a>
+                                </li>
+                                {% elif output.image %}
```

Comment:
I think we're missing the last case of `output.value`

Reply:
I wasn't sure if we wanted that in here. Should it always be fully visible here? Or should it be truncated and fully shown in a popup in case the value is very long?

Reply:
Come to think of it, it might be nice to always show it in a popup, so we can use the same type of links we use for files and images.

Reply:
It's up to you, was only thinking of consistency with the inputs. Actually this is probably fine, we need to really think how to do this given the result templating is only for results json. Maybe just truncate the values for now, revisit it later.